src/components/main/actionsPanel/nodeTreeView/hooks/useCmdk.ts

```diff
@@ -2,19 +2,17 @@ import { useEffect } from "react";
 
 import { isAddNodeAction, isRenameNodeAction } from "@_node/helpers";
 import { useAppState } from "@_redux/useAppState";
-
-import { useNodeActionHandlers } from "./useNodeActionHandlers";
+import { AddNodeActionPrefix, RenameNodeActionPrefix } from "@_constants/main";
 import useRnbw from "@_services/useRnbw";
-import { AddNodeActionPrefix } from "@_constants/main";
-import useFormatCode from "../../../../../actions/useFormatCode.action";
+import useTurnInto from "@_actions/useTurnInto.actions";
```

Comment:
Let's have all these internal actions exported from an index.ts and then imported here

Reply:
Fixed